appcues/src/main/java/com/appcues/data/model/RenderContext.kt

```diff
@@ -0,0 +1,13 @@
+package com.appcues.data.model
+
+internal sealed class RenderContext {
+    object Modal : RenderContext()
+    data class Embed(val frameId: String) : RenderContext()
+
+    override fun toString(): String {
```

Comment:
Not sure if these are the values we want here, we can change later and make sure this matches iOS